launch/launch/utilities/type_utils.py

```diff
@@ -231,6 +231,10 @@ def coerce_to_type(
     :return: `value` coerced to `data_type`.
     """
     def convert_as_yaml(value, error_msg):
+        # Forward empty strings
+        if not value:
```

Comment:
I wondering if this need to match what the comment suggests. This condition will be true for various other kind of input, like `0`, `None`, an empty list/set/dict?

Reply:
Nvm, the function argument is supposed to be a string.

Reply:
If you look at line 251, it is ensured that value is a `str` object.